tests/run/cpp_static_method_overload.pyx

```diff
@@ -0,0 +1,49 @@
+# mode: run
+# tag: cpp, cpp11
```

Comment:
Do this need cpp11? This should work on any version I think?

Reply:
It doesn't -- removed.